app/controllers/JourneyAnswersController.scala

```diff
@@ -358,4 +359,14 @@ class JourneyAnswersController @Inject() (
           }
       }
     }
+
+  def saveForeignPropertiesInformation(taxYear: TaxYear, incomeSourceId: IncomeSourceId, nino: Nino): Action[AnyContent] =
```

Comment:
Move it to new controller?

Reply:
saveForeignPropertiesInformation --> saveSelectCountrySection ?

Reply:
Renamed.